src/main/java/frc/trigon/robot/subsystems/arm/ArmConstants.java

```diff
@@ -0,0 +1,112 @@
+    private static final CANcoder ANGLE_ENCODER = new CANcoder(ANGLE_ENCODER_ID);
+    private static final TalonSRX ELEVATOR_ENCODER = new TalonSRX(ELEVATOR_ENCODER_ID);
+    private static final double
+            ANGLE_MOTOR_OFFSET = 0,
+            ELEVATOR_MOTOR_OFFSET = 0;
+    private static final SensorDirectionValue ANGLE_MOTOR_DIRECTION = SensorDirectionValue.Clockwise_Positive;
+    private static final boolean ELEVATOR_MOTOR_PHASE = true;
+    private static final AbsoluteSensorRangeValue ANGLE_MOTOR_RANGE = AbsoluteSensorRangeValue.Signed_PlusMinusHalf;
+    private static final int
+            ANGLE_P = 0,
+            ANGLE_I = 0,
+            ANGLE_D = 0,
+            ELEVATOR_P = 0,
+            ELEVATOR_I = 0,
+            ELEVATOR_D = 0;
+    private static final PIDController PID_CONTROLLER_ANGLE = new PIDController(ANGLE_P, ANGLE_I, ANGLE_D);
+    private static final PIDController PID_CONTROLLER_ELEVATOR = new PIDController(ELEVATOR_P, ELEVATOR_I, ELEVATOR_D);
+
+    static {
+        configureAngleMotors();
+        configureElevatorMotors();
+        configureAngleEncoder();
+        configureElevatorEncoder();
+    }
+
+    private static void configureAngleMotors() {
+        MASTER_ANGLE_MOTOR.restoreFactoryDefaults();
+        FOLLOWER_ANGLE_MOTOR.restoreFactoryDefaults();
+        MASTER_ANGLE_MOTOR.setInverted(MASTER_ANGLE_MOTOR_INVERTED);
+        FOLLOWER_ANGLE_MOTOR.setInverted(FOLLOWER_ANGLE_MOTOR_INVERTED);
+        MASTER_ANGLE_MOTOR.setIdleMode(ANGLE_MOTORS_IDLE_MODE);
+        FOLLOWER_ANGLE_MOTOR.setIdleMode(ANGLE_MOTORS_IDLE_MODE);
+        MASTER_ANGLE_MOTOR.enableVoltageCompensation(VOLTAGE_COMPENSATION_SATURATION);
+        FOLLOWER_ANGLE_MOTOR.enableVoltageCompensation(VOLTAGE_COMPENSATION_SATURATION);
+    }
+
+    private static void configureElevatorMotors() {
+        MASTER_ELEVATOR_MOTOR.restoreFactoryDefaults();
+        FOLLOWER_ELEVATOR_MOTOR.restoreFactoryDefaults();
+        MASTER_ELEVATOR_MOTOR.setInverted(MASTER_ELEVATOR_MOTOR_INVERTED);
+        FOLLOWER_ELEVATOR_MOTOR.setInverted(FOLLOWER_ELEVATOR_MOTOR_INVERTED);
+        MASTER_ELEVATOR_MOTOR.setIdleMode(ELEVATOR_MOTORS_IDLE_MODE);
+        FOLLOWER_ELEVATOR_MOTOR.setIdleMode(ELEVATOR_MOTORS_IDLE_MODE);
+        MASTER_ELEVATOR_MOTOR.enableVoltageCompensation(VOLTAGE_COMPENSATION_SATURATION);
+        FOLLOWER_ELEVATOR_MOTOR.enableVoltageCompensation(VOLTAGE_COMPENSATION_SATURATION);
+    }
+
+    private static void configureAngleEncoder() {
+        CANcoderConfiguration config = new CANcoderConfiguration();
+        config.MagnetSensor.MagnetOffset = ANGLE_MOTOR_OFFSET;
+        config.MagnetSensor.SensorDirection = ANGLE_MOTOR_DIRECTION;
+        config.MagnetSensor.AbsoluteSensorRange = ANGLE_MOTOR_RANGE;
+        ANGLE_ENCODER.getConfigurator().apply(config);
+    }
+
+    private static void configureElevatorEncoder() {
```

Comment:
You didn't [select the remote encoder](https://v5.docs.ctr-electronics.com/en/stable/ch14_MCSensor.html?highlight=configselected#software-select-sensor)